src/sensor/ping.cpp

```diff
@@ -103,9 +103,8 @@ Ping::Ping() : Sensor()
     // Load last successful connection
     auto config = SettingsManager::self()->lastLinkConfiguration();
     qCDebug(PING_PROTOCOL_PING) << "Loading last configuration connection from settings:" << config;
-    if(!detector()->checkLink(config)) {
-        detectorThread()->start();
-    }
+    tryConnectLink(config);
```

Comment:
This is a regression, no?

The configuration here is appended to the list to detect, not guaranteed to be checked first. We want to check this link explicitly first before detection process starts.